keyboards/cospad/keymaps/detrus/keymap.c

```diff
@@ -0,0 +1,369 @@
+#include QMK_KEYBOARD_H
+#include "led.h"
+#include <print.h>
+
+#ifdef RGBLIGHT_ENABLE
+#include "rgblight.h"
+#endif
+
+#define _______ KC_TRNS
```

Comment:
KC_TRNS is already pre-defined to be seven spaces. You don't need to add this in.

Reply:
Yeah, bet for newer users, it makes it easier to remember, so I don't personally see a problem with this (aside from it just redefining things)